compiler/runtime.go

```diff
@@ -644,15 +644,91 @@ func (e *unaryEvaluator) Type() semantic.MonoType {
 	return e.t
 }
 
+func doUnary(mt semantic.MonoType, op ast.OperatorKind, v values.Value) (values.Value, error) {
+	if op == ast.ExistsOperator {
+		return values.NewBool(!v.IsNull()), nil
+	}
+
+	// If the value is null, return it immediately.
+	if v.IsNull() {
+		return v, nil
+	}
+
+	switch op {
+	case ast.AdditionOperator:
+		// Do nothing.
+		return v, nil
```

Comment:
I always wonder why this operator exists... at least the implementation is efficient...

Reply:
Yeah, I mean... :shrug:

I'm just trying to stay faithful to the source material :rofl: